Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/core/storage/RecoveryCertificateContainer.kt

```diff
@@ -86,6 +86,10 @@ data class RecoveryCertificateContainer(
             override val fullNameFormatted: String
                 get() = certificate.nameData.fullNameFormatted
 
+            override val fullNameStandardizedFormatted: String
+                get() = "${certificate.personIdentifier.lastNameStandardized}," +
```

Comment:
for the name formatting we already have this logic:
```kotlin
val fullNameFormatted: String
            get() = when {
                firstName.isNullOrBlank() -> lastName
                else -> "$lastName, $firstName "
            }
```
we should probably consider also here that the `firstName` could be empty or null and adjust